Userland/Games/Chess/Engine.h

```diff
@@ -38,6 +38,15 @@ class Engine : public Chess::UCI::Endpoint {
         m_bestmove_callback = move(callback);
     }
 
+    void start_new_game()
+    {
+        if (!m_connected)
+            return;
```

Comment:
suspicious check… what about using `VERIFY` instead? 🤔

Reply:
If `m_connected` is false at this point, then either the connection to the engine has been lost, or it has not yet been established. The alternative to returning here would be to call `connect_to_engine_service()`, which would spawn a new engine process for us to send commands to. I think returning is better in this case, because sending what is essentially a reset command to a newly spawned engine won't have any effect. `Engine::quit()` behaves the same way for the same reason.